### ISSUE TITLE
[BUG] Transaction date overlaps Token name in History

### ISSUE THREAD

**Issue by zenkertA** [casinocoin/tokenwallet-mobile#173]
**Describe the bug**
The transaction date in History overview overlaps the Account amount if there is more than 2 figures. See screendump below.

![image](https://user-images.githubusercontent.com/39646311/74159158-91abba00-4c1b-11ea-84b2-551358972961.png)

**To Reproduce**
1. Go to 'History'

**Expected behavior**
Not overlapping

**Suggestion **
Only show Date Hour and Minute not Second in overview or move it below the amount.

**Smartphone (please complete the following information):**
 - Device: Honor 8 lite
 - OS: Android
 - OS Version: 8.0.0
 - App Version: 0.1.15

**Comment by t3nshi** [comment]
I don't see as an error. As the info I would need tho browse the transactions is there. If you think that it should be different, what do you have in mind? I would like it that way so I can see a first glance some info about the transaction to recognize it, I won't mind the Account ID cut off, or the full figure of the amount, but I would not like to cover either the date, nor the token or direcction of transaction, any other info we can see upon entering into details. What do you suggest or think about it?

**Comment by zenkertA** [comment]
Still thinking . . . why display the Account at all?

Why not just the Amount on the the first row and Timestamp on the second row?
The rest is in Details. You don´t read the account anyway. It is an overview.
Plus or Minus, Amount and Timestamp is enough. I´s to techie IMHO.

Another thing reagrding the way it is now.
Why not when it´s an outgoing transaction, show the recieving account. Where the funds have been sent to? And accordingly the text then should be "To: c5Xd6Gh . . . . ".
Feels far more natural than showing what account you sent from.

**Comment by t3nshi** [comment]
I think you are on the right direction. Let's explore a bit more... what info we really need at first glance... And what info we can leave for detailed view.

**Comment by zenkertA** [comment]
Yes let's do that. I also have some thoughts on detailed view which I have
not shared yet.

Den mån 10 feb. 2020 18:29TeNsHi SaMa <notifications@github.com> skrev:

**Comment by zenkertA** [comment]
Hi, I made two suggestions for History/Overview.
The first one is a tweak of the way it looks like today with the addition that if a transaction has occured with one of your Contacts it is shown. I also shrinked the right white spacing before the content a bit since it is a larger than the spacing to the left after the Coin/Token symbol.

![bild](https://user-images.githubusercontent.com/39646311/74252522-19fa8f80-4cee-11ea-9e0d-8035d49afd9b.png)


The second one is different and with less information.

![bild](https://user-images.githubusercontent.com/39646311/74252569-2979d880-4cee-11ea-91af-7f795a75555c.png)


( I have not made anything yet regarding History/details yet. IMHO there is room for a lot of improvements there, will continue with that)

**Comment by t3nshi** [comment]
I'll try to get more people and the team's attention here.

**Comment by zenkertA** [comment]
Done some thinking and sketching in History/details - first round.
I have used the current layout and look and moved stuff around quite a lot
to get what I consider a better flow and read/understanding of what is presented.
For the time being I have 3 comments , see screendump below. Btw I shrinked the Copy buttons a bit. 

Rank 1:
To begin with it starts with the Transaction date, which is natural as this is a part
what is presented at the intitial page of History. It´s a sign of recognition.
(One can consider to move "Amount" to Rank 1 too and move "Date of transaction" to Rank 2.)

Rank 2: 
Comment 1:
Is AccountID field necessary? It has a corresponding field with the same account
information in " From: " where the Account is shown.
Better show the "Account Label" instead of the account and change name to "Account Label"
like it is in the Wallet.
I have also suggested as a Feature that it would be a nice feature for the user to be able
to change the Account Label. Then this field would make perfect sense.
The way it is now it´s just "double" information. The info you get is the same as in - " From: "
Might as well.

Rank 3:
I moved "Amount" to rank 3 as that is a at first glanze very vital info to get.

Rank 4: The transaction account details " From: " and " To: ".
Comment 2:
One can argue if " From: " is needed or is it because of what I wrote in Comment 1 that
I have a hard time coping with it.

Rank 5:
Now comes the nitty gritty details in falling Rank order.
Nothing much to say really, except for Comment 3.
Is that - Minus sign - really needed?

![image](https://user-images.githubusercontent.com/39646311/74358219-61058500-4dc1-11ea-8153-6841708561ad.png)
